Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN pnpm install
 
 COPY ["ui/", "."]
 
-RUN NODE_ENV='production' pnpm run build
+RUN NODE_ENV='production' VERSION=${VERSION} pnpm run build
```

Comment:
For this to work, you need to include `ARG VERSION` somewhere around line 37. (i.e. Between `FROM` and `RUN`)

Reply:
Oh right, the UI builder is separate... thank you